tests/integration/targets/vmware_folder_template_from_vm/vars.yml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
The file with variables for real vcenter should be in `defaults/main.yml` and the variables for the simulator in vars.yml

Reply:
ah ok. is there an existing example of this? i tried looking at the other roles but they all seemed a bit different so i think im getting confused. Seems like `vmware_appliance_info` does what your talking about?

Reply:
yes, `vmware_appliance_info` does this, and actually I think all other integration targets running in upstream/downstream CI (all the ones prefixed with `vmware_`)